### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SunglassJerry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,18 +41,19 @@ Saikou is crafted based on simplistic yet state-of-the-art elegance. It is an [A
...
-|                [Gogo](https://gogoanime.cl)                |   WORKING   |   SOME    |
-|                  [Zoro](https://zoro.to)                   |   WORKING   |    NO     |
-|                 [Marin](https://marin.moe)                 |   WORKING   |    YES    |
-|             [AnimePahe](https://animepahe.ru)              |   WORKING   |    YES    |
-|           [ConsumeBili](https://www.bilibili.tv)           |   WORKING   |    NO     |
-|               [Haho (18+)](https://haho.moe)               |   WORKING   |    YES    |
-|         [HentaiMama (18+)](https://hentaimama.io)          |   WORKING   |   SOME    |
-|       [HentaiStream (18+)](https://hentaistream.com)       | NOT-WORKING |    YES    |
-|           [HentaiFF (18+)](https://hentaiff.com)           |   WORKING   |   SOME    |
+|                            SITE                            |   STATUS    | DIRECT DOWNLOADS |
```
Comment (scrazzz) [inline README.md:44]:
```suggestion
|                            SITE                            |   STATUS    | DOWNLOADS |
```

This change is unnecessary.

**Review by scrazzz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SunglassJerry) [reply README.md:44]: I'd like for Shivam to review it. Technically we can download from all sources using adm/1dm. This column literally indicates sources that support videos that can downloaded directly.

Comment (scrazzz) [reply README.md:44]: I was talking about the commit diffs below this line. The changes you made here have no use whatsoever.

Comment (scrazzz) [reply README.md:44]: Lines 68 and 69 is fine. I don't see the point of changing `DOWNLOADS` to `DIRECT DOWNLOADS`.

Comment (SunglassJerry) [reply README.md:44]: This "Downloads" column indicates if that source support direct 1-tap downloads . It makes more sense to name it Direct Downloads. 
Fact is, we can download from all sources using download managers.

Meh... Let's wait for Shivam. Call it my opinion.

Comment (brahmkshatriya) [reply README.md:44]: I thinks its ok, but idk, it doesnt really matter to the oblivious users

**Pull request merged**